Protocol.md

```diff
@@ -106,7 +106,7 @@ When using a transport protocol that does not provide compatible framing, the Fr
     +-----------------------------------------------+
 ```
 
-* __Frame Length__: (24 = max 16,777,216 bytes) Length of Frame. Excluding Framing Length Field.
+* __Frame Length__: (24 bits = max value 16,777,215) Positive integer representing the length of Frame in bytes. Excluding Framing Length Field.
```

Comment:
unsigned integer? 
(grep the spec for "integer", and make it explicit whether it's signed or unsigned)